hip/static/styles/icons.scss

```diff
@@ -127,3 +127,17 @@ To select that icon, do:
     content: fa-content($fa-var-external-link-alt);
   }
 }
+
+// ------------------ Health Alert Priority icon colors --------- //
+  .update-hip {
+    color: #3a833c;
```

Comment:
If we think we might use these in other places, we could move these to colors.scss (something I should have done in my original PR). Or we could wait to see if they are actually used elsewhere. Up to you.

Reply:
I'll add the colors to `_colors`, but keep the classes here.